SwiftUIBestPractices.md

```diff
@@ -0,0 +1,119 @@
+# SwiftUI Best Practices
+## File Order
+
+### `body` placed after `init`
+
+``` swift
+/// Displays the settings related to voice activation.
```

Comment:
Add some flavor text here on why we do the init before the body. eg. predictable spot, init the the start, the body is the main chunk